cerberus/__init__.py

```diff
@@ -1,7 +1,12 @@
-__all__ = ['client', 'user_auth', 'aws_auth', 'util']
+# Setting up logging here because this is the root of the module
```

Comment:
I could probably move this to the bottom of the file.  But the examples I've seen for this put it up above the rest of the statements in `__init__.py`

Reply:
Nevermind.  Reorganized it.